tests/SprykerEcoTest/Zed/AmazonPay/Business/AmazonpayFacadeAbstractTest.php

```diff
@@ -115,7 +115,7 @@ protected function createShipmentMethod($name, $price)
         $shipmentCarrier->save();
 
         $shipmentMethod = SpyShipmentMethodQuery::create()
-            ->filterByShipmentMethodKey($name)
+            ->filterByName($name)
```

Comment:
Because of this change tests for all previous tests will fail, @pyvovarov-s did you think about this?